frontend/src/routes/HomePage.js

```diff
@@ -28,6 +28,7 @@ class HomePage extends React.Component<{
         const { auth, redirectAuthToDashboard } = this.props;
         if (auth.isAuthenticated && redirectAuthToDashboard) {
             this.props.history.push('/worksheets?name=dashboard');
+            // this.props.history.push('/dashboard');
```

Comment:
Make sure this is uncomment when ready:p

Reply:
Maybe add a TODO comment here?